src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -20,4 +25,28 @@ public LineResponse saveLine(LineRequest request) {
         Line persistLine = lineRepository.save(request.toLine());
         return LineResponse.of(persistLine);
     }
+
+    public LinesResponse getLines() {
+        List<Line> lines = lineRepository.findAll();
+        return new LinesResponse(lines);
+    }
+
+    public LineResponse getLine(Long id) throws NotFoundException {
+        Optional<Line> persistLine = lineRepository.findById(id);
+        return LineResponse.of(persistLine.orElseThrow(NotFoundException::new));
```

Comment:
```suggestion
        return lineRepository.findById(id)
                        .map(LineResponse::of)
                        .orElseThrow(NotFoundException::new);
```

와 같이 작성해보실 수 있을 것 같아요 :)

Reply:
리뷰 감사합니다. 이런 스타일로 바꾸는걸 lambda를 적용했다고 표현하면 될까요?
보기가 훨씬 깔끔하고 가독성이 좋아지네요

Reply:
넵 ㅎㅎ 
정확한 이름은 메서드 레퍼런스라고 합니다!